website/config/webpack.config.dev.js

```diff
@@ -181,7 +181,7 @@ module.exports = {
 
           // Custom CodeMirror loader for syntax highlighting
           {
-            test: /code\/[^/]+$/,
+            test: /code[/\\\\]/,
```

Comment:
This is not an equivalent regex. One will match e.g. "code/FixedSizeGridChildrenClass.js" and the other will match e.g. "code/"

Reply:
Hm. I guess there are other places in the file using the `[/\\\\]` expression though.